cypress/e2e/polls/polls.spec.ts

```diff
@@ -157,6 +157,91 @@ describe("Polls", () => {
         });
     });
 
+    it("Polls without votes can be edited from context menu", () => {
```

Comment:
fwiw, test naming is typically in the following form:

```js
describe("TheTheThingBeingTested", () => {
  it("should have a particular behaviour", () => { ... });
});
```

for example, this test would be something like:

```js
describe("Polls", () => {
  it("should be editable if they have no votes", () => { ... });
});
```

in the grand scheme of things it doesn't really matter - just spreading the word for eventual consistency :)

Reply:
Actually that was my understanding, too. I just followed the style used in the other test methods in this file. I've switched all of them over to the idiomatic format now.